### DISCUSSION TITLE
update changed zoom value

### DISCUSSION THREAD

**Discussion by hdrodel** [inocan-group/vue3-google-map#277] [Q&A] [answered]
Is it possible get the updated zoom value ? does the component emit it?

I have a use case where i want to display certain markers based on how far it's zoomed in. I wanted to use v-if based on zoom value but changing the zoom does not update the zoom parameter. Is there a way to do this? 

Edit:
I've tried this:
```
<GoogleMap 
    api-key: "<key>"
    :zoom="zoom"
   @update:zoom="zoom=$event"
/>
```
but it does not work

**Comment by HusamElbashir** [comment, marked as answer]
The correct name for the event is `zoom_changed`. See here for all the events you can listen to on the map:
https://github.com/inocan-group/vue3-google-map/blob/develop/src/components/GoogleMap.vue#L10-L33
https://developers.google.com/maps/documentation/javascript/reference/map#Map-Events

**Reply by hdrodel** [reply]
I got it working using the zoom_changed event. Could not find it in the vue3-google-map documentation. Feels like a bit of a workaround when vue3 has the option to emit updated values from components but it works! Thank You

**Reply by HusamElbashir** [reply]
We tried to make the event names match those from the Google Maps API for consistency's sake

**Reply by hdrodel** [reply]
May i [suggest](https://github.com/inocan-group/vue3-google-map/pull/278) mentioning the option to listen to the events on the GoogleMap component in the documentation?
